pwndbg/commands/context.py

```diff
@@ -1003,13 +1010,13 @@ def register_context_default(self, reg):
         return f"{prefix} {desc}"
 
 
-def get_regs(regs: List[str] = None):
-    regs: List[Any] = regs
-    result = []
-    rc = RegisterContext()
+def get_regs(in_regs: List[str | VisitableRegister | None] | None = None):
+    if in_regs is None:
+        in_regs = []
```

Comment:
```suggestion
def get_regs(in_regs: List[str | VisitableRegister | None] = []):
```

Maybe this makes more sense?

Reply:
Good call, making this change

Reply:
Oh nevermind - since we add contents to this list, and Python default parameters are always the same instance of the list, this will break (future calls don't get a fresh empty list)

Reply:
ahh you're right, my bad!

Reply:
could maybe add a comment about this